docs/misc/dvd-remux.md

```diff
@@ -0,0 +1,670 @@
+
+                ![MakeMKV output folder](./img/dvd-remux/makemkv/output.png)
+
+            === "Physical disc"
+
+                Insert your physical disc.
+                MakeMKV will automatically detect it,
+                and then show you this window:
+
+                ![MakeMKV main window](./img/dvd-remux/makemkv/inserted-disc-window.png)
+
+                Click the "Backup" button,
+                select "Decrypt video files"
+                (if possible),
+                pick an output directory,
+                and press "OK".
+                It will now start creating
+                a back-up of your disc.
+
+                ![MakeMKV backup window](./img/dvd-remux/makemkv/copy-dvd-window.png)
+
+                Once it's done,
+                [follow the instructions
+                for the ISO method](#__tabbed_5_1).
+
+## Setting the correct aspect ratio
+
+DVDs (and later SD Blu-rays)
+are stored in a format known as "anamorphic video".
+This means that the stored video dimensions
+differ from the intended display dimensions.
+A typical NTSC DVD stores video at 720x480 pixels
+(a 3:2 aspect ratio),
+but displays it at either widescreen (16:9)
+or standard definition (4:3) aspect ratios.
+
+To achieve the correct display size,
+DVDs make use of a [Sample Aspect Ratio (SAR)](https://en.wikipedia.org/wiki/Sample_aspect_ratio),
+also known as a [Pixel Aspect Ratio (PAR)](https://en.wikipedia.org/wiki/Pixel_aspect_ratio).
+This value tells the player how much to stretch each pixel.
+The system was designed for CRT televisions,
+which had [overscan](https://en.wikipedia.org/wiki/Overscan),
+meaning they would stretch and slightly crop
+the edges of the image,
+which allowed the image to reach its intended [Display Aspect Ratio (DAR)](https://en.wikipedia.org/wiki/Display_aspect_ratio).
+
+At the start of the digital age,
+DVD authors accounted for this
+by keeping important content
+within a smaller [active area](https://en.wikipedia.org/wiki/Overscan#Overscan_amounts).
+When played on a CRT,
+the television's natural stretching and cropping
+would result in the correct final aspect ratio
+being displayed.
+Modern displays,
+however,
+lack both overscan
+and the automatic stretching of CRTs.
+As a result,
+DVDs will display incorrectly
+if we don't correct for this
+during the remux process.
+
+Due to the above,
+any DVD remux
+that does not properly account for SAR/DAR
+should be considered ***broken***.
+
+### Heuristics
+
+SAR values have evolved
+through multiple standards over time,
+making it challenging to identify
+the exact standard used for any given disc.
+To help narrow down the correct values,
+a reference table of common SAR values
+and their corresponding active areas
+is provided below[^sar-source]:
+
+[^sar-source]: A number of SAR values were derived from [this page](https://web.archive.org/web/20140218044518/http://lipas.uwasa.fi/~f76998/video/conversion/#conversion_table).
```

Comment:
Similarly, I'd format this link as `[A Quick Guide to Digital Video Resolution and Aspect Ratio Conversions]`